Sources/Kontur.Rabbitmq/AmqpSubscriptionBuilder.cs

```diff
@@ -0,0 +1,54 @@
+﻿using RabbitMQ.Client;
+using System;
+using System.Threading.Tasks.Dataflow;
+
+namespace Kontur.Rabbitmq
+{
+    public class AmqpSubscriptionBuilder : IAmqpSubscriptionBuilder
+    {
+        private IAmqpSerializer defaultSerializer;
+        private IAmqpRouter router;
+        private IAmqpPropertyBuilder propertyBuilder;
+
+        public AmqpSubscriptionBuilder()
+        {
+            this.defaultSerializer = new SimpleSerializer();
+            this.router = new AmqpRouter();
+            this.propertyBuilder = new AmqpPropertyBuilder();
+        }
+
+        public ISubscriptionTag Build<T>(Bus bus)
+        {
+            IAmqpSerializerFactory serializerFactory = new AmqpSerializerFactory("plain/text", defaultSerializer);
+            AmqpMessageBuilder amqpMessageBuilder = new AmqpMessageBuilder(propertyBuilder, router, serializerFactory);
+
+            TransformBlock<IMessage, AmqpMessage> amqpBuilderBlock =
+                new TransformBlock<IMessage, AmqpMessage>(
+                    (Func<IMessage, AmqpMessage>)amqpMessageBuilder.Build);
+
+            var factory = new ConnectionFactory();
+            factory.Uri = new Uri("amqp://localhost");

```

Comment:
It was intended to use hardcoded value of `localhost`?

Reply:
It example how to build pipes of Dataflow and integrate Kontour with Kontour.RabbitMQ. This method should be refactored aggressively. For example, the connection factory should be incapsulated into a ConnectionPool and a ConnectionPool shoud be shared among all AmqpSubscriptionBuilder instances. This method shoud provide a fluent interface to build subsriptions conveniently.